SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/EventTracingTests/EventTracingTest.cpp

```diff
@@ -147,40 +141,50 @@ void AEventTracingTest::GatherData()
 		return A.CreationTime > B.CreationTime;
 	});
 
+	int RequiredRuntime = 1;
 	int RequiredClients = GetRequiredClients();
 	int RequiredWorkers = GetRequiredWorkers();
+
+	int FoundRuntime = 0;
 	int FoundClient = 0;
 	int FoundWorker = 0;
+
+	bool FoundAllEventTraceLogs = false;
+
 	for (const FileCreationTime& FileCreation : FileCreationTimes)
 	{
-		if (FoundClient != RequiredClients && FileCreation.FilePath.Contains("client"))
+		if (FoundClient != RequiredClients && FileCreation.FilePath.Contains(TEXT("client")))
```

Comment:
Is this case invariant? It works already so I guess it must be

Reply:
Yeah I think it must be too XD